src/modules/drawing/components/DrawingToolboxButton.vue

```diff
@@ -40,9 +49,24 @@ export default {
             }
             return null
         },
+        buttonStyle() {
+            return {
+                // The buttons having different/dynamic widths looks unintentional.
+                // While inline styles are bad, there is no Bootstrap class we can
+                // apply to the component and this width is too use-case specific
+                // to be added as a class in the ButtonWithIcon component.
+                width: this.uiMode === 'MENU_ALWAYS_OPEN' ? '4.5em' : '',
+            }
+        },
```

Comment:
I would prefer that we set a custom class dynamically, and then have this 4.5rem defined as a CSS property for this class (if we can avoid mixing CSS and JS, it's better)

Or specify something in ButtonWithIcon. As you've stated you felt it's too specific to do so, but I wouldn't mind an extra flag to cover this case if it's too complicated with a class/CSS approach (or it requires some !important)

Reply:
I'll see what I can do 👍